app/presenters/licence-versions/view.presenter.js

```diff
@@ -22,12 +22,35 @@ function go(licenceVersion) {
       href: `/system/licences/${licence.id}/history`,
       text: 'Go back to history'
     },
+    changeType: licenceVersion.administrative ? 'no licence issued' : 'licence issued',
+    notes: _notes(licenceVersion),
     pageTitle: `Licence version starting ${formatLongDate(licenceVersion.startDate)}`,
     pageTitleCaption: `Licence ${licence.licenceRef}`,
-    changeType: licenceVersion.administrative ? 'no licence issued' : 'licence issued'
+    reason: _reason(licenceVersion)
   }
 }
 
+function _notes(licenceVersion) {
```

Comment:
If this is the change for B&D users, shouldn't it include the permission checks? The same goes for created on and created by in `_reason()`.

Reply:
Based on the title and description, I'd land on doing them as part of this change, especially as including permissions might result in a different implementation of how to determine what to show.